src/mame/drivers/jalmah.cpp

```diff
@@ -1174,19 +1207,15 @@ static const gfx_layout tilelayout =
 	16,16,
 	RGN_FRAC(1,1),
 	4,
-	{ 0, 1, 2, 3 },
-	{ 0*4, 1*4, 2*4, 3*4, 4*4, 5*4, 6*4, 7*4,
-			16*32+0*4, 16*32+1*4, 16*32+2*4, 16*32+3*4, 16*32+4*4, 16*32+5*4, 16*32+6*4, 16*32+7*4 },
-	{ 0*32, 1*32, 2*32, 3*32, 4*32, 5*32, 6*32, 7*32,
-			8*32, 9*32, 10*32, 11*32, 12*32, 13*32, 14*32, 15*32 },
+	{ STEP4(0,1) },
+	{ STEP8(0,4), STEP8(4*8*16,4) },
+	{ STEP16(0,4*8) },
 	32*32
 };
 
 static GFXDECODE_START( gfx_urashima )
 	GFXDECODE_ENTRY( "scroll0", 0, tilelayout, 0x100, 16 )
-	GFXDECODE_ENTRY( "scroll3", 0, charlayout, 0x000, 16 )
-	GFXDECODE_ENTRY( "scroll1", 0, tilelayout, 0x100, 16 )
-	GFXDECODE_ENTRY( "scroll2", 0, tilelayout, 0x100, 16 )
+	GFXDECODE_ENTRY( "scroll1", 0, charlayout, 0x000, 16 )
```

Comment:
This was left intentionally. 
What's the MACHINE_UNEMULATED_PROTECTION part you don't understand?